tests/test_other.py

```diff
@@ -139,7 +139,7 @@ def get_fastcomp_src_dir():
 
 
 def parse_wasm(filename):
-  wat = run_process([os.path.join(building.get_binaryen_bin(), 'wasm-dis'), filename], stdout=PIPE).stdout
+  wat = shared.run_process([os.path.join(building.get_binaryen_bin(), 'wasm-dis'), filename], stdout=PIPE).stdout
```

Comment:
could these be just `run_process` without `shared.`? I don't think it helps disambiguate the `self.run_process` one, that's already different. better to shorten as much as possible.